label/label_selinux.go

```diff
@@ -104,7 +104,13 @@ func Relabel(path string, fileLabel string, relabel string) error {
 	if fileLabel == "" {
 		return nil
 	}
-	if relabel == "z" {
+	if !strings.ContainsAny(relabel, "z & Z") {
+		return nil
+	}
+	if strings.Contains(relabel, "z & Z") {
```

Comment:
I think this is probably incorrect, unless you are searching for the substring "z & Z" in relabel.